doc/submitter_guide.md

```diff
@@ -271,20 +271,24 @@ Example: `"2020-08-14T23:08:10.008000+00:00"`
 #### Tests
 
 ##### `status`
-The test status string, one of the following.
-
-* `ERROR` - the test is faulty, the status of the tested code is unknown.
-* `FAIL` - the test has failed, the tested code is faulty.
-* `PASS` - the test has passed, the tested code is correct.
-* `DONE` - the test has finished successfully, the status of the tested code
-  is unknown.
-* `SKIP` - the test wasn't executed, the status of the tested code is unknown.
+The test status string, one of the following:
+
+* "FAIL" - the test completed and reported the tested code as faulty.
+* "ERROR" - the test didn't complete due to a failure in its code, and the
+  status of the tested code is unknown.
+* "MISS" - the test didn't run due to a failure in the test harness, and the
```

Comment:
I feel like we could extend this explanation a little bit to include the ideas that justified its inclusion in the list, for example, we should make it clear how `MISS` is not `null` and when `MISS` instead of `ERROR`

Reply:
Expanded on the statuses, in the doc. Thanks!